docs/managed-zone.md

```diff
@@ -72,48 +72,11 @@ EOF
 
-
 ### Current limitations
 At the moment the MGC is given credentials to connect to the DNS provider at startup using environment variables, because of that, MGC is limited to one provider type (Route53), and all zones must be in the same Route53 account.
 
 There are plans to make this more customizable and dynamic in the future, [work tracked here](https://github.com/Kuadrant/multicluster-gateway-controller/issues/228).
 
 ## Spec of a ManagedZone
-The ManagedZone is a simple resource with an uncomplicated API, see a sample [here](../../config/samples/kuadrant.io_v1alpha1_managedzone.yaml).
-
-### Mandatory fields
-The ManagedZone spec has 2 required fields: `domainName` and `dnsProviderSecretRef`:
-
-```yaml
-apiVersion: kuadrant.io/v1alpha1
-kind: ManagedZone
-metadata:
-  name: my-test-aws-zone
-  namespace: multi-cluster-gateways
-spec:
-  domainName: mydomain.example.com
-  description: "My Managed Zone"
-  dnsProviderSecretRef:
-    name: my-aws-credentials
-    namespace: multicluster-gateway-controller-system
-```
+The ManagedZone is a simple resource with an uncomplicated API, see a sample [here](../config/samples/kuadrant.io_v1alpha1_managedzone.yaml).
 
-#### Spec fields
-| Key                    | Example Value                                                             | Required? | Description                                |
-|------------------------|---------------------------------------------------------------------------|-----------|--------------------------------------------|
-| `dnsProviderSecretRef` | `name: my-credential, namespace: multicluster-gateway-controller-system ` | Required  | Ref to DNS Provider Secret                 |
-| `domainName`           | `myapps.example.com`                                                      | Required  | Root Domain Name for this ManagedZone      |
-| `id`                   | `Z0WDADW1234`                                                             | Optional  | Zone ID for an existing Zone in GCP or AWS |
-
-#### Additional notes on spec fields
-
-* `dnsProviderSecretRef` 
-  * This is a reference to a `Secret` that contains a credential for accessing the DNS Provider. See [DNSProvider](dns-provider.md) for more details.
-* `id`
-  * By setting the `id`, you are referring to an existing zone in the DNS provider, which MGC will use to manage the DNS of this zone.
-  * By leaving the `id` empty, MGC will create a zone in the DNS provider, and store the reference in this field.
-* `description`
-  * This is simply a human-readable label/description of this resource (e.g. "Use this zone for the staging environment").
-* `ParentManagedZone`
-  * This allows a zone to be owned by another zone (e.g test.api.domain.com could be owned by api.domain.com), MGC will use this owner relationship to manage the NS values for the subdomain in the parent domain.  
-  * Note that for this to work, both the owned and owner zones must be in the Route53 account accessible by MGC.
```

Comment:
@maleck13 Where do we want this overview doc? I just moved it directly under docs to match what the kuadrant-operator does

Reply:
yeah that is fine. More so we need to add a link to the overview and ref in the docs site nav https://github.com/Kuadrant/docs.kuadrant.io